sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/AzureDigitalTwinsAPIImplBuilder.java

```diff
@@ -64,6 +85,104 @@ public AzureDigitalTwinsAPIImplBuilder serializerAdapter(SerializerAdapter seria
         return this;
     }
 
+    /*
+     * The HTTP client used to send the request.
+     */
+    private HttpClient httpClient;
+
+    /**
+     * Sets The HTTP client used to send the request.
+     *
+     * @param httpClient the httpClient value.
+     * @return the AzureDigitalTwinsAPIImplBuilder.
+     */
+    public AzureDigitalTwinsAPIImplBuilder httpClient(HttpClient httpClient) {
+        this.httpClient = httpClient;
+        return this;
+    }
+
+    /*
+     * The configuration store that is used during construction of the service
+     * client.
+     */
+    private Configuration configuration;
+
+    /**
+     * Sets The configuration store that is used during construction of the service client.
+     *
+     * @param configuration the configuration value.
+     * @return the AzureDigitalTwinsAPIImplBuilder.
+     */
+    public AzureDigitalTwinsAPIImplBuilder configuration(Configuration configuration) {
+        this.configuration = configuration;
+        return this;
+    }
+
+    /*
+     * The logging configuration for HTTP requests and responses.
+     */
+    private HttpLogOptions httpLogOptions;
+
+    /**
+     * Sets The logging configuration for HTTP requests and responses.
+     *
+     * @param httpLogOptions the httpLogOptions value.
+     * @return the AzureDigitalTwinsAPIImplBuilder.
+     */
+    public AzureDigitalTwinsAPIImplBuilder httpLogOptions(HttpLogOptions httpLogOptions) {
+        this.httpLogOptions = httpLogOptions;
+        return this;
+    }
+
+    /*
+     * The service API version that is used when making API requests.
+     */
+    private ServiceVersion serviceVersion;
+
+    /**
+     * Sets The service API version that is used when making API requests.
+     *
+     * @param serviceVersion the serviceVersion value.
+     * @return the AzureDigitalTwinsAPIImplBuilder.
+     */
+    public AzureDigitalTwinsAPIImplBuilder serviceVersion(ServiceVersion serviceVersion) {
```

Comment:
This is interesting. It looks like this new autorest version lets us set the service API version instead of hardcoding it to the version in the swagger we generated it from. This will come in handy for later